### ISSUE TITLE
Missing certificate for the Pubmatic endpoint

### ISSUE THREAD

**Issue by moteus** [prebid/prebid-server#1190]
When i try tu make request to the new pubmatic enpoint i get error like.
`Post https://hbopenbid.pubmatic.com/translator?source=prebid-server: x509: certificate signed by unknown authority`
To fix it i just add `certificate_file` pointed to the system ca-bundle file

**Comment by bastianb-yieldlove** [comment]
Could you elaborate on how you solved this? We see the same problem, but setting the config variable seems to do nothing.

**Comment by SyntaxNode** [comment]
Did you symlink `certificate_file` to the system bundle?

**Comment by moteus** [comment]
I set `certificates_file: "/etc/ssl/certs/ca-certificates.crt"` in the pbs.yaml file.

**Comment by bastianb-yieldlove** [comment]
I got it. We run an ECS cluster and stupid me was using the name of the crt file on the instance, and not the one from the container.

**Comment by moteus** [comment]
Can you please confirm that this is bug or not.
If not I just close the issue

**Comment by PubMatic-OpenWrap** [comment]
Hi team, 
There is a hardcoded list of certs in prebid-server that is pretty old. So, anyone who is not setting the `certificate_file` property would end up using this hardcoded list. For bidders having secure end-points configured with newer certificates, this could result in certificate errors like the one mentioned above.
I've raised a PR: #1201 where I have updated the hardcoded list of certs to the latest list for Ubuntu. This should fix the error you're getting while calling the PubMatic secure end-point.
